packs/spells/lift-natures-caul.json

```diff
@@ -21,7 +24,7 @@
+            "value": "<p>With a wave of the hand, you cause the targets to see the world around them as if a gauzy film had been lifted away to expose the truth that lies below their perceived reality. Natural features they can see grow twisted and horrendous, structures appear distorted and ruined, and objects seem warped and alien. This alternate vision fades quickly, but the glimpse beyond the natural world lingers for the spell's duration. Aberrations or worshippers of Outer Gods or Great Old Ones are bolstered by this vision of warped reality and gain a +1 status bonus to Will saving throws and attack rolls for 1 minute @UUID[Compendium.pf2e.spell-effects.Item.ihv1azg80N3kj7Vo]{Spell Effect: Lift Nature's Caul (Bonus)}, while all other creatures must attempt a Will saving throw.</p>\n<hr />\n<p><strong>Critical Success</strong> The creature is unaffected and temporarily immune for 1 hour.</p>\n<p><strong>Success</strong> The creature is @UUID[Compendium.pf2e.conditionitems.Item.Frightened]{Frightened 1}.</p>\n<p><strong>Failure</strong> The creature is @UUID[Compendium.pf2e.conditionitems.Item.Stupefied]{Stupefied 1} for 1 minute and frightened 1.</p>\n<p><strong>Critical Failure</strong> The creature is stupefied 1 for 1 minute and @UUID[Compendium.pf2e.conditionitems.Item.Frightened]{Frightened 2}. It can't reduce the value of its frightened condition below 1 as long as it remains stupefied by this spell.</p>\n<hr />\n<p><strong>Heightened (+2)</strong> You increase the area by 5 feet and the duration of the stupefied condition by 1 minute.</p>"
```

Comment:
```suggestion
            "value": "<p>With a wave of the hand, you cause the targets to see the world around them as if a gauzy film had been lifted away to expose the truth that lies below their perceived reality. Natural features they can see grow twisted and horrendous, structures appear distorted and ruined, and objects seem warped and alien. This alternate vision fades quickly, but the glimpse beyond the natural world lingers for the spell's duration. Aberrations or worshippers of Outer Gods or Great Old Ones are bolstered by this vision of warped reality and gain a +1 status bonus to Will saving throws and attack rolls for 1 minute @UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Lift Nature's Caul (Bonus)], while all other creatures must attempt a Will saving throw.</p>\n<hr />\n<p><strong>Critical Success</strong> The creature is unaffected and temporarily immune for 1 hour.</p>\n<p><strong>Success</strong> The creature is @UUID[Compendium.pf2e.conditionitems.Item.Frightened]{Frightened 1}.</p>\n<p><strong>Failure</strong> The creature is @UUID[Compendium.pf2e.conditionitems.Item.Stupefied]{Stupefied 1} for 1 minute and frightened 1.</p>\n<p><strong>Critical Failure</strong> The creature is stupefied 1 for 1 minute and @UUID[Compendium.pf2e.conditionitems.Item.Frightened]{Frightened 2}. It can't reduce the value of its frightened condition below 1 as long as it remains stupefied by this spell.</p>\n<hr />\n<p><strong>Heightened (+2)</strong> You increase the area by 5 feet and the duration of the stupefied condition by 1 minute.</p>"
```